tests/tests/say_bubbles.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { test, expect } from "@playwright/test";
+import { getPage } from "./utils/auth";
+import { publicTestMapUrl } from "./utils/urls";
+
+test.describe("Say bubbles", () => {
+    // test.beforeEach(async ({ page }) => {
+
+    // });
+
+    test("should display a speech bubble when sending a message", async ({ browser }) => {
+
+        const page = await getPage(browser, 'Alice',
+            publicTestMapUrl("tests/E2E/empty.json", "say_bubbles")
+        );
```

Comment:
Ideally, you should open 2 browsers here (Alice and Bob).
You should check that bob receives the message when Alice talks too.